src/daemon/http/server.rs

```diff
@@ -572,8 +577,24 @@ pub async fn metrics(req: Request) -> RoutingResult {
 
                     res.push('\n');
                     res.push_str(
-                    "# HELP krill_ca_child_last_connection unix timestamp in seconds of last child to CA connection\n",
-                );
+                        "# HELP krill_ca_child_state child state (see 'suspend_child_after_inactive_hours' config) (0=suspended, 1=active)\n",
+                    );
+                    res.push_str("# TYPE krill_ca_child_state gauge\n");
+                    for (ca, status) in ca_status_map.iter() {
+                        // skip the ones for which we have no status yet, i.e it was really only just added
```

Comment:
This loop doesn't seem to skip anything.

Reply:
removed comment